modules/Makefile.am

```diff
@@ -47,6 +47,7 @@ include modules/systemd-journal/Makefile.am
 include modules/system-source/Makefile.am
 include modules/tagsparser/Makefile.am
 include modules/xml/Makefile.am
+include modules/regexp/Makefile.am
```

Comment:
maybe we should rename the directory to regexp-parser, regexps are used in a lot of ways within syslog-ng and not all of them would concentrate in the "modules/regexp" directory (neither now or long term), so let's try to use a more specific directory.